apps/models.go

```diff
@@ -50,7 +50,7 @@ type Select struct {
 	Placeholder string `yaml:"placeholder,omitempty" json:"placeholder,omitempty"`
 
 	// Default value will be the value for this field, before user interaction. Having a default value will override the placeholder text
-	DefaultValue string `yaml:"default,omitempty" json:"default,omitempty"`
+	DefaultValue string `yaml:"default_value,omitempty" json:"defaultValue,omitempty"`
```

Comment:
Why?

Reply:
https://github.com/ketch-com/rest/blob/main/server/models.go#L2973 and https://github.com/ketch-com/rest/blob/main/server/models.go#L3007. Thoughts on best key name - happy to update rest

Reply:
Why can't we keep it as `default` in the YAML file at least? Or is this not related to that?

Reply:
We can